automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/notebooks/NotebookGCEClusterMonitoringSpec.scala

```diff
@@ -0,0 +1,145 @@
+package org.broadinstitute.dsde.workbench.leonardo.notebooks
+
+import java.io.File
+
+import org.broadinstitute.dsde.workbench.auth.AuthToken
+import org.broadinstitute.dsde.workbench.leonardo._
+import org.broadinstitute.dsde.workbench.leonardo.rstudio.RStudio
+import org.scalatest.tagobjects.Retryable
+import org.scalatest.{DoNotDiscover, ParallelTestExecution}
+
+import scala.concurrent.duration._
+import scala.util.Try
+
+/**
+ * This spec verifies cluster status transitions like pause/resume and cluster PATCH.
+ * It is similar in intent to ClusterStatusTransitionsSpec but uses notebooks for validation,
+ * so lives in the notebooks sub-package.
+ */
+@DoNotDiscover
+class NotebookGCEClusterMonitoringSpec extends GPAllocFixtureSpec with ParallelTestExecution with NotebookTestUtils {
```

Comment:
I would rename anything that says `cluster` here to `runtime` since we're using VMs for all the GCE tests, as opposed to using clusters in the Dataproc tests